### ISSUE TITLE
List/Document keyboard nav tasks from the Tool

### ISSUE THREAD

**Issue by mwroffo** [mwroffo/asdf#43]
<p>List/Document keyboard nav tasks from the Tool, these should be within the AC of the Epic and if not to document them in case of future epic(accessibility) </p>
